scripts.py

```diff
@@ -296,44 +297,6 @@ def open(self):
         return StringIO(representation.content)
 
 
-class LaneSweeperScript(LibraryInputScript):
```

Comment:
The part of this code that really needs to be in circulation (because it deals with setting up a fake app server) has been moved to `CacheRepresentationPerLane`.